tfx/examples/cifar10/cifar10_utils_native_keras.py

```diff
@@ -0,0 +1,338 @@
+# Lint as: python2, python3
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Python source file includes CIFAR10 utils for Keras model.
+
+The utilities in this file are used to build a model with native Keras.
+This module file will be used in Transform and generic Trainer.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from typing import List, Text
+
+import os
+import absl
+import tensorflow as tf
+import tensorflow_transform as tft
+
+import flatbuffers
+# pylint: disable=g-direct-tensorflow-import
+from tflite_support import metadata as _metadata
```

Comment:
Did you do a pip install for tflite_support to work? If so, then @zhitaoli , how do we include this as part of the tfx release?

Reply:
This is example code, so we could create a setup.py similar to https://github.com/tensorflow/tfx/blob/master/tfx/examples/custom_components/presto_example_gen/setup.py